src/sagemaker_xgboost_container/algorithm_mode/train.py

```diff
@@ -117,6 +117,12 @@ def get_validated_dmatrices(
 
     return train_dmatrix, val_dmatrix, train_val_dmatrix
 
+# TODO: Add data distributionType check here.
+def eligible_for_dask_run(tree_method_selected: str, num_gpus: int, is_pipe: bool, content_type: str) -> bool:
```

Comment:
Where do we call this?

Reply:
In line 185 below. I commented out the actual call, till we're ready to enable the usage.